plenum/server/node.py

```diff
@@ -1818,7 +1818,11 @@ def processOrdered(self, ordered: Ordered):
             logger.trace("{} got ordered requests from backup replica {}".
                          format(self, inst_id))
             r = False
-        self.monitor.requestOrdered(req_idrs, inst_id, byMaster=r)
+        if inst_id < self.instances.count:
+            self.monitor.requestOrdered(req_idrs, inst_id, byMaster=r)
+        else:
+            logger.debug('{} got ordered request for instance {} which '
```

Comment:
`self.instances.count` will decrease when a node is suspended; value of `f` changes, hence no. of instances decrease.